### PR TITLE
Feature/mds 329132 replace pump asset example app

### PR DESCRIPTION
**Pull request by StefanKellerSiemens**
With this pull request we like to introduce two changes
- Updating description to reflect changes to Mendix V9 theme concept
- Replaced Pump Asset Example with the Operation Insight as this shows more the power of Mendix and MindSphere

Please assign this branch to Mark van Ments and do not merge without further notice as our new theme is not yet available in the App Store.

@Mark: can you please do a review

Many thanks

### CODE REVIEW

**Review by MarkvanMents — COMMENTED** [review]
@StefanKellerSiemens,
I've finished reviewing these changes. I've done some proofreading, but they all look good.

I've made one suggestion, about describing the app roles as a table in the text as well as the picture. This would make it more accessible, but it is up to you.

Let me know when you are ready to publish this.

Mark

**Comment by MarkvanMents** [comment]
Note to self (@MarkvanMents )
Consider updating content/partners/siemens/attachments/mindsphere-module-details/image2.png to show Mx9.6+ version of the explorer.

File: content/partners/siemens/mindsphere-example-app.md [hunk 1]
```diff
@@ -33,85 +34,104 @@ You need a MindSphere user account on a tenant including a Developer Cockpit - e
...
 
-{{% alert type="info" %}}
-If you have more then one tenant, with an Developer Cockpit option, on MindSphere you will get a list of those tenants. Choose the tenant where you want to register your app.
+	![Choose app registration](attachments/mindsphere-example-app/autoreg-web-key.png)
 
-If you have only one tenant on MindSphere, the process will automatically select this tenant for you.
-{{% /alert %}}
+	{{% alert type="info" %}}If you have more then one tenant with a Developer Cockpit option, MindSphere will present you with a list of those tenants. Choose the tenant in which you want to register your app.<br/><br/>If you have only one tenant on MindSphere, the process will automatically select this tenant for you.{{% /alert %}}
+
+3. Give a **DIsplay Name**, **Internal Name**, and, optionally, a **Description** in order to register your app.
```
Comment (HeikoOsigus) [inline content/partners/siemens/mindsphere-example-app.md:87]: DIsplay --> Display

File: content/partners/siemens/mindsphere-module-details.md [hunk 2]
```diff
@@ -326,7 +326,7 @@ and directly after the script of the X-XRSR put the following script
 ```
 
 {{% alert type="info" %}}
-IF you do not use the **Siemens MindSphere Web Content** module you have to create the mindspherelogin.html by your own in the folder /theme/web. See the [mindspherelogin.html](#mindspherelogin) section, below.
+IF you do not use the **Siemens MindSphere Web Content** module you have to create the mindspherelogin.html yourself in the folder /theme/web. See the [mindspherelogin.html](#mindspherelogin) section, below.
```
Comment (HeikoOsigus) [inline content/partners/siemens/mindsphere-module-details.md:329]: IF --> If

File: content/partners/siemens/mindsphere-example-app.md [hunk 3]
```diff
@@ -1,275 +1,163 @@
...
-MindSphere SSO provides the user’s tenant as the **Name** attribute in the **Tenant** entity.
+	A new tab opens showing the registration detail of your app.
+	
+6. Click **Configure** to add the required MindSphere roles.
 
-{{% image_container width="75%" %}}![Tenant and Tenant Object Domain Model](attachments/mindsphere-example-app/image12.png){{% /image_container %}}
+	![Choose app registration](attachments/mindsphere-example-app/devcockpit-app-details.png)
 
-By utilizing this value when an entity is accessed, the Mendix app can be made multi-tenant. Note that security has to be applied explicitly in the action which accesses the entity, it cannot be applied to the entity itself.
+7. Add the following MindSphere API Roles to your app and assign them to correct application roles **admin** and **user**, as shown in the picture below:
```
[the remark this answers is not in the document]
Comment (HeikoOsigus) [reply content/partners/siemens/mindsphere-example-app.md]: Created a table

**Review by MarkvanMents — APPROVED** [review]
Looks great.
Thanks for collaboration.
Release approved over email.

**Pull request merged**